great_expectations/expectations/expectation.py

```diff
@@ -1771,7 +1772,7 @@ class ColumnAggregateExpectation(BatchExpectation, ABC):
         InvalidExpectationConfigurationError: If no `column` is specified
     """  # noqa: E501
 
-    column: str
+    column: str = Field(description=COLUMN_FIELD_DESCRIPTION)
```

Comment:
No problem adding this here, but you will need to do this with the rest of the fields next.